docs/03-apis/api-v2/editing-values.md

```diff
@@ -14,208 +14,202 @@ HTTP POST to http://host/v2/values
+    "anything": "http://0.0.0.0:3333/ontology/0001/anything/v2#"
   }
 }
 ```
 
-Each value can have an optional custom IRI (of [Knora IRI](knora-iris.md#iris-for-data) form) specified by the `@id` attribute, 
-a custom creation date specified by adding `knora-api:valueCreationDate` (an [xsd:dateTimeStamp](https://www.w3.org/TR/xmlschema11-2/#dateTimeStamp)), 
-or a custom UUID given by `knora-api:valueHasUUID`. Each custom UUID must be [base64url-encoded](rfc:4648#section-5), without padding. 
-If a custom UUID is provided, it will be used in value IRI. If a custom IRI is given for the value, its UUID should match 
-the given custom UUID. If a custom IRI is provided, but there 
-is no custom UUID provided, then the UUID given in the IRI will be assigned to the `knora-api:valueHasUUID`. 
-A custom value IRI must be the IRI of the containing resource, followed
-by a `/values/` and a custom ID string. For example: 
+Each value can have an optional custom IRI (of [Knora IRI](knora-iris.md#iris-for-data) form) specified by the `@id`
+attribute, a custom creation date specified by adding `knora-api:valueCreationDate` (an
+[xsd:dateTimeStamp](https://www.w3.org/TR/xmlschema11-2/#dateTimeStamp)), or a custom UUID given by
+`knora-api:valueHasUUID`. Each custom UUID must be [base64url-encoded](rfc:4648#section-5), without padding. If a custom
+UUID is provided, it will be used in value IRI. If a custom IRI is given for the value, its UUID should match the given
+custom UUID. If a custom IRI is provided, but there is no custom UUID provided, then the UUID given in the IRI will be
+assigned to the `knora-api:valueHasUUID`. A custom value IRI must be the IRI of the containing resource, followed by
+a `/values/` and a custom ID string. For example:
 
-
-```jsonld
+```json
   "@id" : "http://rdfh.ch/0001/a-thing",
-  "@type" : "anything:Thing",
-  "anything:hasInteger" : {
-    "@id" : "http://rdfh.ch/0001/a-thing/values/IN4R19yYR0ygi3K2VEHpUQ",
-    "@type" : "knora-api:IntValue",
-    "knora-api:intValueAsInt" : 21,
-    "knora-api:valueHasUUID" : "IN4R19yYR0ygi3K2VEHpUQ",
-    "knora-api:valueCreationDate" : {
-        "@type" : "xsd:dateTimeStamp",
-        "@value" : "2020-06-04T12:58:54.502951Z"
-      }
-  },
-  "@context" : {
-    "knora-api" : "http://api.knora.org/ontology/knora-api/v2#",
-    "anything" : "http://0.0.0.0:3333/ontology/0001/anything/v2#",
-    "xsd" : "http://www.w3.org/2001/XMLSchema#"
-  }
+"@type": "anything:Thing",
+"anything:hasInteger": {
+"@id" : "http://rdfh.ch/0001/a-thing/values/IN4R19yYR0ygi3K2VEHpUQ",
+"@type": "knora-api:IntValue",
+"knora-api:intValueAsInt": 21,
+"knora-api:valueHasUUID": "IN4R19yYR0ygi3K2VEHpUQ",
+"knora-api:valueCreationDate": {
+"@type": "xsd:dateTimeStamp",
+"@value": "2020-06-04T12:58:54.502951Z"
+}
+},
+"@context": {
+"knora-api": "http://api.knora.org/ontology/knora-api/v2#",
+"anything": "http://0.0.0.0:3333/ontology/0001/anything/v2#",
+"xsd": "http://www.w3.org/2001/XMLSchema#"
+}
```

Comment:
indentions are lost ;)

Reply:
yes, because it wasn't valid JSON, so Intellij didn't like it. Fixed now, thx